assets/templates/users/new.html

```diff
@@ -80,5 +77,25 @@ <h1>Shelter Registration</h1>
 
         return false;
     };
+
+    var toggleRequiredFormFields = function () {
+        var streetDiv = document.getElementById('shelterStreet');
+        var cityDiv = document.getElementById('shelterCity');
+        var stateDiv = document.getElementById('shelterState');
+        var shelterZipDiv = document.getElementById('shelterZip');
+        var type = Number(document.getElementById('createForm').elements.namedItem('type').value);
+
```

Comment:
Have you considered doing a CSS class toggle instead of doing them individually?